.github/workflows/deploy-zeit-staging.yml

```diff
@@ -41,10 +41,15 @@ jobs:
           ZEIT_DEPLOYMENT_URL=`echo $ZEIT_DEPLOYMENT_OUTPUT | egrep -o 'https?://[^ ]+.now.sh'`
           echo "::set-env name=ZEIT_DEPLOYMENT_URL::$ZEIT_DEPLOYMENT_URL"
 
-          ZEIT_DEPLOYMENT_ALIAS=$(cat now.json | jq -r '.name')-${CURRENT_BRANCH##*/}.now.sh
+          if [[ ${CURRENT_BRANCH##*/} =~ ^v[0-9]{1,}- ]]; then # Checking if pattern matches with "vX-" where X is a number
+            ZEIT_DEPLOYMENT_ALIAS=${CURRENT_BRANCH##*/}
+          else
+            ZEIT_DEPLOYMENT_ALIAS=$(cat now.json | jq -r '.name')-${CURRENT_BRANCH##*/}
+          fi
+          ZEIT_DEPLOYMENT_ALIAS=$(echo $ZEIT_DEPLOYMENT_ALIAS | head -c 45).now.sh # Zeit alias only allow 45 characters, so we are taking the first 53 characters of this alias
```

Comment:
@Demmonius 45 vs  53? (doc issue)

Reply:
Yes I should explain it

Reply:
Yes please, I don't really understand it, as it